### ISSUE TITLE
Review Page: Leaving with ranking 0 should be possible

### ISSUE THREAD

**Issue by FelixPetriconi** [ACCUConf/Submission_Web_Application#88]
I think it should be possible to leave the review page of a proposal without setting the ranking above 0. I often like to leave a comment for later coming back. 
Without this change, I would have to set all proposals that I want to rank later to 1, but then I cannot use the jump to the next unrated proposal feature, which is very useful.

**Comment by russel** [comment]
A score of 0 is abstain, it isn't an actual score. Does this change the scenario?

**Comment by FelixPetriconi** [comment]
Yes, even when I (temporarily) abstain from a score, it might make sense to store a comment for the committee or in my case, to come back later.

**Comment by russel** [comment]
OK. We need to put that scenario into a test and ensure it works.

We will need to then decide if comment to the proposer for an abstain are still sent.

**Comment by FelixPetriconi** [comment]
From my point of view, comments from such abstained proposals should not be sent out.
